ddtrace/tracer/tracer.go

```diff
@@ -45,6 +46,13 @@ type tracer struct {
 	// stopped is a channel that will be closed when the worker has exited.
 	stopped chan struct{}
 
+	// stopping synchronizes calls to Stop() to ensure the tracer is stopped
+	// exactly once.
+	stopping sync.Mutex
+
+	// wg is used to wait for the worker goroutines to exit before Stop() returns.
```

Comment:
```suggestion
	// wg waits for all goroutines to exit before Stop() returns.
```
Nit: I think `worker` maybe doesn't fully apply to all goroutines started and maybe "is used" can be omitted.

Reply:
👍 Agreed. The previous comment was poorly worded.